int64 GC=0
int64 PC_q=1
int64 PC_X=2
int64 DC=3

int64 control_type

int64 RBDL=0
int64 Pin=1

int64 control_lib

float64 coeff_cmd